### PR TITLE
[Supporting] Resharper 8.2

### PR DESCRIPTION
**Pull request by michaelmairegger**

### CODE REVIEW

**Comment by JakeGinnivan** [comment]
Awesome! I have been meaning to do this. Thanks!

I will look and merge/release tonight after work :)

File: Build.cmd [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 set framework=v4.0.30319
 
-"%SystemDrive%\Windows\Microsoft.NET\Framework\%framework%\MSBuild.exe" %~dp0\XamlAttributeOrdering.sln /Property:Configuration=Release
+"%SystemDrive%\Windows\Microsoft.NET\Framework\%framework%\MSBuild.exe" "%~dp0\XamlAttributeOrdering.sln"/Property:Configuration=Release
```
Comment (JakeGinnivan) [inline Build.cmd:5]: There is a space missing here (not going to stop me merging)

**Comment by JakeGinnivan** [comment]
I still had issues building after I merged this, but got them sorted and 1.2.2 has been released with 8.2 support.

Thanks for getting the ball rolling

**Comment by michaelmairegger** [comment]
:+1: thanks

**Pull request merged**